esx_service/utils/auth_api.py

```diff
@@ -28,6 +28,7 @@
 
 # regex for valid tenant name
 VALID_TENANT_NAME_REGEXP = "[a-zA-Z0-9_][a-zA-Z0-9_.-]*"
+VALID_TENANT_NAMES = 'rename of vm-groups other than _DEFAULT'
```

Comment:
The error msg comes out like this,
Vm-group name _DEFAULT is invalid, only _rename of vm-groups other than _DEFAULT_ is allowed